go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	golang.org/x/sys v0.2.0
 	google.golang.org/grpc v1.51.0
-	k8s.io/client-go v0.26.0-rc.0
+	k8s.io/client-go v0.27.0-alpha.0
 	k8s.io/klog/v2 v2.80.1
-	k8s.io/kubelet v0.26.0-rc.0
+	k8s.io/kubelet v0.27.0-alpha.0
```

Comment:
@pohly can you check if dependabot can ignore a regex? Perhaps alpha versions like this?

Reply:
I think @jsafrane  checked this, and if I remember this wouldn't happen if this repo hadn't been updated to use a rc version.